cgd_tx_eff/src/edu/ohsu/compbio/txeff/tx_eff_hgvs.py

```diff
@@ -426,8 +427,11 @@ def _lookup_variant_hgvs_transcripts(self, variant: Variant, pysam_file):
                 self.logger.warning(f"Invalid variant interval {variant}: %s", str(e))
                 self._benchmark_cancel()
             except HGVSDataNotAvailableError as e:
-                # HGVS uses this exception for unrecoverable connection errors, and for when a row in the database just wasn't found.                
-                if 'alignment' in str(e).lower():
+                # HGVS uses this exception for unrecoverable connection errors, and for when a row in the database just wasn't found.
+                if not re.match('^NM_[0-9]+\.[0-9]+$', refseq_transcript):
```

Comment:
Should this regular expression match other types of RefSeq sequences (NR, NP, XM, XR, XP)? Also it might be useful to put an example of the message you intend to match against in a comment here.

Reply:
That is a good question. 

The query for transcripts (line 336 self.hdp.get_tx_for_region) only returns transcripts with the NM and NR prefix.  I just confirmed this by running the query on my local uta installation.

Transcripts with the NR do cause an exception to be thrown but they are caught in a different place: 
* See comment on line 356: ``Non-coding transcripts (prefix NR_) cause HGVS to throw an exception when determining c. and p.``
* See comment on line 361: ``# Determine c. and p.. Non coding transcripts will throw an error. See HGVSUsageError caught below.``
* The call to``self.am.c_to_p`` throws an exception when it gets an NR 
* See line 411 and 412: Special handling for the NR transcript.

Reply:
Updated comment.